### PR TITLE
Revert "changes robocop law priority"

### PR DESCRIPTION
**Pull request by ghost**
validhunting is bad

### CODE REVIEW

**Comment by tkdrg** [comment]
> *No description provided.*

**Comment by Cheridan** [comment]
> *No description provided.*

**Comment by Chnkr** [comment]
> *No description provided.*
Oh?

**Comment by Incoming5643** [comment]
> <em>Dead or alive, you're coming with me</em>

**Comment by Cheridan** [comment]
Your move, creep.

**Comment by CPTANT** [comment]
Is this trying to imply the original lawset isn't played as valid the salid?

**Comment by optimumtact** [comment]
At least the original one wasn't quite so clearly orientated towards it

**Comment by tkdrg** [comment]
I don't really like how new robocop is being played in-game. It ends up being really vague since each player has their own interpretation of "space law". It's also more security/command aligned than the original. Idk, I think it's even worse than valadin.

**Comment by Iamgoofball** [comment]
Like I said in the original PR:

stop trying to turn all the funny lawsets into asimov #382139472394729347

:+1: to this PR

**Comment by CPTANT** [comment]
@tkdlrg
Are you saying people have a more vague interpretation of space law than of "serve the public trust"?

I don't get it, all complaints mention things that are even more prevalent in the original sequence.

**Comment by tkdrg** [comment]
They're both vague in different ways. "Serve the public trust" is vague because "public trust" isn't defined anywhere, so it ends up being interpreted as something like "be nice"/Asimov Law 2 most of the time. "Uphold the law" is vague because it turns the incomplete guideline known as Space Law into a binding rule and leads to a ton of inane arguments with command/admins about what's written on the wiki page is supposed to mean which isn't enjoyable for any of the sides involved.

**Comment by Anonmare** [comment]
IRC The Public Trust is an actual thing and, ingame-wise, basically means make sure security isn't being corrupt and acting outside of their authority (Like demanding all-access or perma-brigging the clown for honking without a permit). 
The fact it was highest in priority meant the Officers had to understand they were under scrutiny as well if the AI was even halfway competent. 
Plus it's a fucking Robocop reference, changing the priority ruins the joke

**Comment by CPTANT** [comment]
Ok, I can't say I particularly  care any more.

**Comment by Cheridan** [comment]
whatever

File: code/game/objects/items/weapons/AI_modules.dm [hunk 1]
```diff
@@ -354,9 +354,9 @@ AI MODULES
 /obj/item/weapon/aiModule/core/full/robocop
 	name = "'Robo-Officer' Core AI Module"
 	origin_tech = "programming=4"
-	laws = list("Uphold the law.",\
-				"Protect the innocent.",\
-				"Serve the public trust.")
+	laws = list("Serve the public trust.",\
+				"Protect the innocent",\
```
Comment (Cheridan) [inline code/game/objects/items/weapons/AI_modules.dm]: you removed a period :^)

Comment (duncathan) [inline code/game/objects/items/weapons/AI_modules.dm]: UNREPORTED CHANGES WOOOOOW

Comment (Chnkr) [inline code/game/objects/items/weapons/AI_modules.dm]: Go yell at @CPTANT i just used the github revert button.

**Comment by CPTANT** [comment]
There was no period originally, so I added it for consistency

**Pull request merged**